src/task/spawn_local.rs

```diff
@@ -0,0 +1,32 @@
+use std::future::Future;
+
+use crate::task::{Builder, JoinHandle};
+
+/// Spawns a task onto the thread-local executor.
+///
+/// This function is similar to [`std::thread::spawn`], except it spawns an asynchronous task.
+///
+/// [`std::thread`]: https://doc.rust-lang.org/std/thread/fn.spawn.html
```

Comment:
This should be dropped, `spawn_local` has no equivalent in the thread module.

Reply:
fixed